packages/terra-date-time-picker/tests/wdio/date-time-picker-mobile-spec.js

```diff
@@ -13,39 +9,75 @@ Terra.describeViewports('Date Time Picker Twelve Hour Mobile', ['tiny'], () => {
   });
 
   describe('Date Time Picker Twelve Hour Mobile - Default with no time', () => {
-    before(() => browser.url('/#/raw/tests/terra-date-time-picker/date-time-picker/date-time-picker-12-hour-mobile'));
+    before(() => browser.url('/#/raw/tests/terra-date-time-picker/date-time-picker/twelve-hour/date-time-picker-empty-12-hour-mobile'));
 
-    Terra.it.validatesElement({ axeRules: { rules: ignoredA11y } });
+    Terra.it.validatesElement();
   });
 
   describe('Date Time Picker Twelve Hour Mobile - Clicking Meridiem updates the time', () => {
     it('Date Time Picker Twelve Hour Mobile - Clicking Meridiem updates the time', () => {
-      browser.url('/#/raw/tests/terra-date-time-picker/date-time-picker/date-time-picker-12-hour-mobile');
+      browser.url('/#/raw/tests/terra-date-time-picker/date-time-picker/twelve-hour/date-time-picker-value-12-hour-mobile');
 
       browser.click('//*[@id="datetimeValueProvided"]/div[2]/div/div[3]/button[1]');
-      Terra.validates.element({ axeRules: { rules: ignoredA11y } });
+      Terra.it.validatesElement();
 
       browser.click('//*[@id="datetimeValueProvided"]/div[2]/div/div[3]/button[2]');
     });
   });
 
-  describe('Date Time Picker Twelve Hour Mobile - Focus Hour Styles', () => {
+  describe('Date Time Picker Twelve Hour Value Mobile - Focus Hour Styles', () => {
     before(() => {
-      browser.url('/#/raw/tests/terra-date-time-picker/date-time-picker/date-time-picker-12-hour-mobile');
+      browser.url('/#/raw/tests/terra-date-time-picker/date-time-picker/twelve-hour/date-time-picker-value-12-hour-mobile');
 
       browser.click('//*[@id="terra-time-hour-input"]');
     });
 
-    Terra.it.validatesElement({ axeRules: { rules: ignoredA11y } });
+    Terra.it.validatesElement();
+  });
+
+  describe('Date Time Picker Twelve Hour Empty Mobile - Focus Hour Styles', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-date-time-picker/date-time-picker/twelve-hour/date-time-picker-empty-12-hour-mobile');
+
+      browser.click('//*[@id="terra-time-hour-input"]');
+    });
+
+    Terra.it.validatesElement();
+  });
+
+  describe('Date Time Picker Twelve Hour Disabled Mobile - Focus Hour Styles', () => {
```

Comment:
Nothing is being focused in this test

Reply:
Focus Hours changed to disabled hours in 009e0a3